pkg/storaged/format-dialog.jsx

```diff
@@ -202,10 +210,22 @@ function format_dialog_internal(client, path, start, size, enable_dos_extended,
     extract_option(split_options, "noauto");
     const opt_ro = extract_option(split_options, "ro");
     const opt_never_auto = extract_option(split_options, "x-cockpit-never-auto");
+    const opt_nofail = extract_option(split_options, "nofail");
+    const opt_netdev = extract_option(split_options, "_netdev");
     const extra_options = unparse_options(split_options);
 
     let existing_passphrase_type = null;
 
+    let at_boot;
+    if (opt_never_auto)
+        at_boot = "never";
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17888-20221124-142402-f4c856ed-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)